ckanext/dgu/bin/fix_secondary_themes_2.py

```diff
@@ -0,0 +1,83 @@
+'''
+Fix the multiple JSON encoding of the secondary theme extra
+
+e.g.
+
+"[\"Health\"]" -> ["Health"]
+'''
+
+import os
+import sys
+from sqlalchemy import engine_from_config
+from pylons import config
+import common
+import ast
+import json
+from optparse import OptionParser
+from ckan import model
+
+from running_stats import StatsList
+
+stats = StatsList()
+
+
+class FixSecondaryTheme2(object):
+    @classmethod
+    def command(cls, config_ini, write):
+        common.load_config(config_ini)
+        common.register_translator()
+
+        rev = model.repo.new_revision()
+        rev.author = 'fix_secondary_theme.py'
```

Comment:
Can you prefix the author with "script-"? Otherwise the edits will show up in the publisher activity reports.